test/js/bun/http/bun-serve-static.test.ts

```diff
@@ -143,7 +145,7 @@ describe("static", () => {
         Bun.gc(true);
 
         const rss = (process.memoryUsage.rss() / 1024 / 1024) | 0;
-        expect(rss).toBeLessThan(baseline * 4);
```

Comment:
This test was not failing when it should've been failing. The test itself shouldn't use a ton of memory so I think a hardcoded value is fine here.